### PR TITLE
Update bptc and rgtc tests with level and dimension test

### PR DESCRIPTION
**Pull request by shrekshao**
Updating test according to recent change: https://github.com/KhronosGroup/WebGL/pull/3111

addressing chrome issue: http://crbug.com/1100834

### CODE REVIEW

File: sdk/tests/js/tests/compressed-texture-utils.js [hunk 1]
```diff
@@ -192,6 +192,28 @@ var testTexSubImageDimensions = function(gl, validFormats, expectedByteLength, g
     gl.deleteTexture(tex);
 };
 
+var testTexImageLevelDimensions = function(gl, validFormats, expectedByteLength, getBlockDimensions, imageConfigs) {
```
Comment (kenrussell) [inline sdk/tests/js/tests/compressed-texture-utils.js:195]: Might be a good time to start changing "var" to "let", at least inside this function.

**Comment by shrekshao** [comment]
> has this been tested with revised validation code in ANGLE and/or Chromium?
The level validation already passed on Win (using D3D angle).
Not yet on Mac (current chrome fails test). Updated validation is still compiling (once run out of disk storage). I will post updates

**Comment by shrekshao** [comment]
FYI. Local test on Mac Chromium shows updated validation code passed the new test.

**Comment by kenrussell** [comment]
Thanks for confirming. Merging now.

**Pull request merged**